web/src/components/FolderListView.tsx

```diff
@@ -42,6 +42,7 @@ const FolderListView: React.FC<FolderListViewProps> = ({
     }, [auth.user, folders]);
 
     useEffect(() => {
+        // must be useCallback, otherwise it will rerender constantly. enforce?
```

Comment:
I like to add `NOTE: ` to things like this to make it clear its extra information, not just explaining the code

Reply:
Will follow this convention :thumbsup: